src/mlj_embedder_interface.jl

```diff
@@ -0,0 +1,128 @@
+### EntityEmbedder with MLJ Interface
+
+# 1. Interface Struct
+mutable struct EntityEmbedder{M <: MLJFluxModel} <: Unsupervised
+    model::M
+end;
+
+
+# 2. Constructor
+function EntityEmbedder(model;)
+    return EntityEmbedder(model)
+end;
+
+
+# 4. Fitted parameters (for user access)
+MMI.fitted_params(::EntityEmbedder, fitresult) = fitresult
+
+# 5. Fit method
+function MMI.fit(transformer::EntityEmbedder, verbosity::Int, X, y)
+    return MLJModelInterface.fit(transformer.model, verbosity, X, y)
+end;
+
+
+# 6. Transform method
+function MMI.transform(transformer::EntityEmbedder, fitresult, Xnew)
+    Xnew_transf = MLJModelInterface.transform(transformer.model, fitresult, Xnew)
+    return Xnew_transf
+end
+
+# 8. Extra metadata
+MMI.metadata_pkg(
+    EntityEmbedder,
+    package_name = "MLJTransforms",
+    package_uuid = "23777cdb-d90c-4eb0-a694-7c2b83d5c1d6",
+    package_url = "https://github.com/JuliaAI/MLJTransforms.jl",
+    is_pure_julia = true,
+)
+
+MMI.metadata_model(
+    EntityEmbedder,
+    input_scitype = Table,
+    output_scitype = Table,
+    load_path = "MLJTransforms.EntityEmbedder",
+)
+
+MMI.target_in_fit(::Type{<:EntityEmbedder}) = true
+
+
+
+
+
+"""
+$(MMI.doc_header(EntityEmbedder))
+
+`EntityEmbedder` implements entity embeddings as in the "Entity Embeddings of Categorical Variables" paper by Cheng Guo, Felix Berkhahn.
+
+# Training data
+
+In MLJ (or MLJBase) bind an instance unsupervised `model` to data with
+
+    mach = machine(model, X, y)
+
+Here:
+
+
+- `X` is any table of input features (eg, a `DataFrame`). Features to be transformed must
+   have element scitype `Multiclass` or `OrderedFactor`. Use `schema(X)` to 
+   check scitypes. 
+
+- `y` is the target, which can be any `AbstractVector` whose element
+  scitype is `Continuous` or `Count` for regression problems and 
```

Comment:
```suggestion
  scitype is `Continuous` or for regression problems and 
```

`Count` targets are not currently supported by the MLJFlux models you can wrap here.

Perhaps to prevent tragile wrapper docs, you can just say that `X` and `y` can have any type supported by the model  being wrapped?

Reply:
Done in 8ac77e4ed2cce4869dcdf98a8ebd57634cd47f06.